web/gds-user-ui/src/components/Section/NetworkActivity/__tests__/NetworkActivity.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import axios from "axios";
+import { mockNetworkActivityData } from "../_mocks_";
+import { networkActivity } from "../service";
+import { render, screen } from "utils/test-utils";
+import NetworkActivity from "../NetworkActivity";
+
+function renderComponent() {
+    return render(<NetworkActivity />);
+}
+
+describe('Network Activity Service', () => {
+    it('returns network service data with response', async () => {
+        const data = mockNetworkActivityData;
+        axios.get = jest.fn().mockResolvedValue({ data });
+        await expect (networkActivity()).resolves.toEqual(data);
+    });
+});
+
+describe('Network Activity Component', () => {
+    it('should render', () => {
+        const { container } = renderComponent();
+        expect(container).toMatchSnapshot();
+    });
+
+/*     it('should display the correct label on the y-axis', () => {
```

Comment:
This will be updated once the component is made visible again. I'll create a follow on story to replace the mock data for the component and then I'll update the test.